src/jit/compiler.h

```diff
@@ -1176,6 +1176,10 @@ struct FuncInfoDsc
     BYTE                unwindCodes[offsetof(UNWIND_INFO, UnwindCode) + (0xFF*sizeof(UNWIND_CODE))];
     unsigned            unwindCodeSlot;
 
+#define MAX_CFICODE_INDEX   0x200
+    BYTE                cfiCodes[(MAX_CFICODE_INDEX * sizeof(CFI_CODE))];
+    unsigned            cfiCodeIndex;
+
```

Comment:
Do we have concern about TP/code size because of this? Or Is it convention to squeeze down the code as possible?